app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str):
+    res = command.split(" ")
+    if res[0] == "cp":
+        with open(res[1], "r") as source:
+            data = source.read()
+
+        with open(res[2], "w") as target:
+            target.write(data)
```

Comment:
you can use like in example  
with open(..., "r") as file_in, open(..., "w") as file_out: